dags/dap_collector_ppa_dev.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+DOCS = """
+### PPA Dev DAP Collector
+
+#### Description
+
+Runs a Docker image that collects PPA Dev Environment data from a DAP (Distributed Aggregation Protocol) leader and stores it in BigQuery.
+
+The container is defined in
+[docker-etl](https://github.com/mozilla/docker-etl/tree/main/jobs/dap-collector-ppa-dev)
+
+This DAG requires following variables to be defined in Airflow:
+* dap_ppa_dev_auth_token
+* dap_ppa_dev_hpke_private_key
+* dap_ppa_dev_task_config_url
+* dap_ppa_dev_ad_config_url
+
+This job is under active development, occasional failures are expected.
+
+#### Owner
+
+bbirdsong@mozilla.com
+"""
+
+default_args = {
+    "owner": "bbirdsong@mozilla.com",
+    "email": ["ads-eng@mozilla.com", "bbirdsong@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2024, 4, 19),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=15),
+}
+
+project_id = "moz-fx-ads-nonprod"
+ad_table_id = "ppa_dev.measurements"
+report_table_id = "ppa_dev.reports"
+
+tags = [
+    Tag.ImpactTier.tier_3,
+    Tag.Triage.no_triage,
+]
+
+
+with DAG(
+    "dap_collector_ppa_dev",
+    default_args=default_args,
+    doc_md=DOCS,
+    schedule_interval="*/15 * * * *",
+    tags=tags,
```

Comment:
You might want to explicitly set `catchup=False` so it doesn't trigger all runs since the start_date/last run if it's not necessary.  False might be the default value but I can't remember
```suggestion
    tags=tags,
    catchup=False,
```

Reply:
Yeah, I can add that.